src/components/ServiceArea.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const ServiceArea = () => {
+  return (
+    <div className='bg-gray-100'>
+      <div className='px-6 pt-10'>
+        <h2 className='text-center mb-3 text-5xl md:text-6xl font-header'>Areas We Serve</h2>
+        <hr className='w-[180px] md:w-[720px] h-px mx-auto mt-2 mb-10 md:mb-2 bg-gray-500 border-0 rounded dark:bg-gray-700' />
```

Comment:
Nitpicking (nb): Just got a comment from our mentor, we have to check with the designer about the underline hr tag, as the length is not using Tailwind prefixes.